app/decorators/search_result_collection_decorator.rb

```diff
@@ -10,19 +10,7 @@ def context
   end
 
   def page
-    if object.page > number_of_pages
-      number_of_pages
-    else
-      object.page
-    end
-  end
-
-  def total_results
-    if object.total_results > result_limit
-      result_limit
-    else
-      object.total_results
-    end
+    object.page + 1
```

Comment:
presuming this is down to zero indexing?

Reply:
Yes.